components/rmrk/Create/CreateCollection.vue

```diff
@@ -185,7 +174,15 @@ export default class CreateCollection extends mixins(
         : api.tx.utility.batchAll
       const args = !this.hasSupport
         ? mintInteraction
-        : [asSystemRemark(api, mintInteraction), ...(await this.canSupport())]
+        : [
+            asSystemRemark(api, mintInteraction),
+            ...(await canSupport(this.hasSupport)),
+          ]
+
+      if (this.base.file) {
+        console.log('[UPLOADING FILE]')
+        uploadDirect(this.base.file, this.accountId).catch(console.warn)
```

Comment:
could we do the catch after `try` block? 
via
```
catch(err) {
...
}
```

Reply:
Yeah that’s the catch. I do not care if its uploaded or not